aunt-leahs-app/src/components/adminLocationList/locationListItem.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ListItem from '@material-ui/core/List';
+import { TextField } from '@material-ui/core';
+import Fab from '@material-ui/core/Fab';
+import ClearIcon from '@material-ui/icons/Clear';
+import '../../App.css';
```

Comment:
I don't think the css file needs to be imported here

Reply:
Yep, tested without and seems to work fine. My vs code didn't grey it out though so I didn't catch it myself 😅